### DISCUSSION TITLE
Can toasts actually be used already? How?

### DISCUSSION THREAD

**Discussion by sceee** [cdmoro/bootstrap-vue-3#1311] [Q&A] [answered]
Hi,

I'm a little confused by #1210 , the docs and https://github.com/bootstrap-vue-next/bootstrap-vue-next/discussions/266.
Is there a simple way to fire toasts at the moment (e.g. using a `show` meethod of `useToast`)?

Does `useToast` currently still require a separate container to render the toasts in? Is there anywhere a documentation how to configure/use it?
And will this configuration be the way-to-go forward?

Or is it best to wait for #1210 to arrive because as far as I understood the conversation over there, it makes the custom container no longer necessary (if I am correct with my assumption).

Thanks!

**Comment by soorajsprakash** [comment]
It doesn't work as of now. 
Dev said he will rewrite the toast part.

In the mean time, you can make ur own toast with css, and show that html element on a condition.

**Comment by xvaara** [comment, marked as answer]
It is usable in the latest releases.
